chart/templates/benchmark-cis-1.8.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   name: cis-1.8
 spec:
   clusterProvider: ""
-  minKubernetesVersion: "1.26.0"
+  minKubernetesVersion: "1.26.0"
```

Comment:
Can we revert the deletion of the last line break please?

Reply:
done